src/Tests/dotnet-new.Tests/Approvals/DotnetVisualBasicClassTemplatesTest.module.langVersion=10.0.targetFramework=net6.0.verified/ClassLib.vbproj

```diff
@@ -0,0 +1,9 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RootNamespace>ClassLib</RootNamespace>
```

Comment:
I would suggest no to have `RootNamespace` defined, except there is a specific reason I'd miss here.
It's likely that folks are renaming this project. But then they would need to change the `RootNamespace` in addition.

Reply:
Oh, I see. My bad.
If they come from the current templates, then we should modify them for .NET 8. As @KathleenDollard said: We are not servicing them for .NET 7.